.gitlab-ci.yml

```diff
@@ -501,15 +533,13 @@ build/amd/openmpi/clang/rocm502/release/shared:
   extends:
     - .build_and_test_template
     - .default_variables
-    - .full_test_condition
-    - .use_gko-rocm502-openmpi-gnu11-llvm11-multi-gpu
```

Comment:
this one would then be unneeded as well I think

Reply:
You mean remove this job entirely ?

Reply:
Yeah, I just kept it in case we want to use that later.